docs/user/index.rst

```diff
@@ -0,0 +1,45 @@
+User Guide
+==========
+
+This section of the documentation provides information about how to install
+uvloop as well as how to use uvloop.
+
+
+Installation
+------------
+
+`uvloop` is available from PyPI. It requires Python 3.5 and Cython.
```

Comment:
uvloop only needs Cython for manual builds. If you install it from PyPI -- only Python 3.5 is needed.